src/main/java/uk/gov/hmcts/reform/sendletter/domain/LetterStatus.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.sendletter.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class LetterStatus {
+
+    @JsonProperty(value = "message_id")
+    public final String messageId;
```

Comment:
Do we just have `messageId ` in the status or we will be sending other fields such as `createdAt` `sentToPrintAt` `printedAt`?

Reply:
will update the domain model once the db layer is merged 😉